components/src/preact/mutationComparison/mutation-comparison.tsx

```diff
@@ -170,7 +170,16 @@ const Toolbar: FunctionComponent<ToolbarProps> = ({
             />
             <CsvDownloadButton
                 className='mx-1 btn btn-xs'
-                getData={() => getMutationComparisonTableData({ content: filteredData }, proportionInterval)}
+                getData={() =>
+                    getMutationComparisonTableData({ content: filteredData }, proportionInterval).map((row) => {
+                        return {
+                            mutation: row.mutation.toString(),
+                            ...Object.entries(row)
+                                .filter(([key]) => key !== 'mutation')
+                                .reduce((acc, [key, value]) => ({ ...acc, [key]: value }), {}),
+                        };
```

Comment:
```suggestion
                        return {
                            ...row,
                            mutation: row.mutation.toString(),
                        };
```

I think this might actually do the same?